R/http_request_error.R

```diff
@@ -3,16 +3,18 @@
 #' @description
 #' Translate a http error code into an error message.
 #'
-#' @param response_status Response code from an API request
+#' @param response HTTP response from the API
```

Comment:
Should we specify that it's a list with a status element?

Reply:
Added